include/circt/Support/SVAttributes.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CIRCT_SUPPORT_SV_ATTRIBUTES_H
+#define CIRCT_SUPPORT_SV_ATTRIBUTES_H
+#include "circt/Support/LLVM.h"
+
+namespace circt {
+bool hasSVAttributes(mlir::Operation *op);
+
+mlir::Attribute getSVAttributes(mlir::Operation *op);
```

Comment:
I think having this outside the SV dialect is required right? Otherwise there might be a cyclic dependency between HW->SV and SV->HW?

I'm not sure where the best place to put this is... Support seems super general, and this is fairly specific. Could this header go into the SV dialect section of the sources (include/circt/Dialect/SV), and the library in the SV dialect section (lib/Dialect/SV), but be a separate library to include so that the cyclic dependence could be avoided?

Reply:
Thanks, yes this separation does make sense. Regarding naming, I'd be fine with `include/Dialect/SV/SVAttributes.h` and `lib/Dialect/SVAttributes.cpp`, and a `CIRCTSVAttributes` lib.

Reply:
Yes, but there is already `SVAttributes.h` which defines SV dialect attributes. Maybe we might want to use different term for SV attributes...